HousingRepairsOnlineApi/UseCases/SaveRepairRequestUseCase.cs

```diff
@@ -3,20 +3,19 @@
 using HousingRepairsOnlineApi.Domain;
 using HousingRepairsOnlineApi.Gateways;
 using HousingRepairsOnlineApi.Helpers;
-using Microsoft.Extensions.Logging;
 
 namespace HousingRepairsOnlineApi.UseCases
 {
     public class SaveRepairRequestUseCase : ISaveRepairRequestUseCase
     {
-        private readonly IRepairStorageGateway cosmosGateway;
+        private readonly IRepairStorageGateway dynamoDBGateway;
```

Comment:
Please use a generic name as these dependancy is on the interface and not a specific implementation.
```suggestion
        private readonly IRepairStorageGateway repairStorageGateway;
```